manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -635,8 +658,8 @@ data:
         {{- end }}
         - name: istio-podinfo
           mountPath: /etc/istio/pod
-        {{- if and (eq .Values.global.proxy.tracer "lightstep") .Values.global.tracer.lightstep.cacertPath }}
-        - mountPath: {{ directory .ProxyConfig.GetTracing.GetLightstep.GetCacertPath }}
+         {{- if and (eq .Values.global.proxy.tracer "lightstep") .Values.meshConfig.defaultConfig.tracing.tlsSettings }}
```

Comment:
nit: Values. meshConfig.defaultConfig -> . ProxyConfig

they represent the same thing but the latter takes into account per pod annotation